include/nbl/builtin/hlsl/bit.hlsl

```diff
@@ -0,0 +1,55 @@
+#ifndef _NBL_BUILTIN_HLSL_BIT_INCLUDED_
+#define _NBL_BUILTIN_HLSL_BIT_INCLUDED_
+
+#include <nbl/builtin/hlsl/cpp_compat.hlsl>
+
+
+#ifndef __HLSL_VERSION
+#include <bit>
```

Comment:
I still need `using ` aliases between std::rotl and `nbl::hlsl` for C++

Reply:
done using this:
https://github.com/Devsh-Graphics-Programming/Nabla/blob/new_hlsl_xoroshiro/include/nbl/builtin/hlsl/cpp_compat.hlsl#L12-L17

Reply:
clever and nice